### DISCUSSION TITLE
Render Preview not displaying

### DISCUSSION THREAD

**Discussion by MarkN-B** [bnpr/Malt#308] [Q&A]
Hello,

I just downloaded Malt for a project. however as you can see in the image, no render preview displays. I am not a coder, so I am unsure if it is a bug, or if I am doing something wrong.
![image](https://user-images.githubusercontent.com/106532243/170987016-1828df3c-fbb3-4972-88c9-e3f4d4b5db1f.png)

I am using blender 3.1.2, and I have openGL 4.6

Any help would be appreciated.

Thank you.

**Comment by pragma37** [comment]
Hi! Yes, this looks like a bug.
Please, open a new report and make sure you include all the required info:
https://github.com/bnpr/Malt/issues/new/choose

**Reply by MarkN-B** [reply]
Thank you, I will.
